packages/v4/src/content/design-guidelines/components/overflow-menu/overflow-menu.md

```diff
@@ -2,4 +2,35 @@
 id: Overflow menu
 section: components
 ---
-An **overflow menu** groups a set of actions in a responsive horizontal list. Actions can be set to persist or collapse into a kebab as the viewport shrinks. Overflow menus are useful in toolbars or other places where a group of actions are required.
+
+An **overflow menu** groups a set of actions in a responsive horizontal list to help declutter an interface. Actions can be set to persist or collapse into a kebab as the viewpoint shrinks. Overflow menus are useful in toolbars or other places where a group of actions is required. 
+
+## Usage 
+Use an overflow menu when additional options are available to the user but there is a space constraint. Common usage for overflow menu's happen when switching a UI from a desktop to a mobile device. 
+
+### Examples
+* Use an overflow menu in a table toolbar to group a number of actions and create visual space. Avoid having more than 3 actions fully displayed within a toolbar.
+
+
+  <img src="./img/toolbaroverflow.png" alt="image showing toolbar overflow" width="1765" />
```

Comment:
I think this build is failing because you don't need the `.` at the beginning of the image src url.

Reply:
Okay wait I just went back into VSC and all the additional content I added is no longer there. not sure why that happened, hahah but I'm assuming that could be part of the problem. Going back now